huaweicloud/resource_huaweicloud_cce_node_v3.go

```diff
@@ -821,9 +821,20 @@ func resourceCCENodeV3Delete(d *schema.ResourceData, meta interface{}) error {
 					return fmtp.Errorf("Error unsubscribing HuaweiCloud CCE node: %s", err)
 				}
 			}
+
+			// wait for the ecs server of the prePaid node to be deleted
+			computeClient, err := config.ComputeV2Client(GetRegion(d, config))
+			if err != nil {
+				return fmtp.Errorf("Error creating HuaweiCloud compute client: %s", err)
+			}
+			pending := []string{"ACTIVE", "SHUTOFF"}
+			target := []string{"DELETED", "SOFT_DELETED"}
+			deleteTimeout := d.Timeout(schema.TimeoutDelete)
+			if err := watiForServerTargetState(computeClient, serverID, pending, target, deleteTimeout); err != nil {
```

Comment:
`watiFor..` suggest to modify the function name in another PR.

Reply:
sure